docs/xss.rst

```diff
@@ -0,0 +1,95 @@
+Performing XSS Vulnerability Audits
+==================================
+
+The bok-choy framework includes the ability to perform XSS (cross-site scripting) audits on
+web pages using a short XSS locator defined in
+https://www.owasp.org/index.php/XSS_Filter_Evasion_Cheat_Sheet#XSS_Locator.
+
+Leverage Your Existing Tests and Fail on XSS Vulnerability Errors
+-------------------------------------------------------------
```

Comment:
FWIW this doc doesn't just live in the repo, it gets processed by Sphinx and uploaded to readthedocs as the official documentation for the package. http://bok-choy.readthedocs.org/